salt/loader.py

```diff
@@ -1265,6 +1268,12 @@ def __init__(
         _generate_module("{}.ext".format(self.loaded_base_name))
         _generate_module("{}.ext.{}".format(self.loaded_base_name, tag))
 
+    def clean_modules(self):
+        for name in list(sys.modules):
+            if name.startswith(self.loaded_base_name):
+                mod = sys.modules.pop(name)
+                del mod
```

Comment:
Can't you just do `sys.modules.pop(name)` without all the `mod =` and `del mod` stuff?